docs/actions.md

```diff
@@ -513,6 +514,35 @@ $ wsk action invoke --blocking --result helloPython --param name World
   }
 ```
 
+### Packaging Python actions in zip files
+
+You can package a Python action and dependant modules in a zip file.
+
+The virtual environment package *virtualenv* is supported to pass pip installed packages together with the action in the zip file.
+Package installations inside virtualenv must be done in the target environment (alpine, python 2.7), so a docker image (e.g. jfloff/alpine-python:2.7) is used to create the virtualenv directory. 
```

Comment:
I believe this approach is putting more power into the hands of the action developer.  She is not _limited_ to the pip modules that are in the docker image but can bring along other pip modules (or newer/older versions) with her code.  In this scenario, she would:
1. Make a virtual environment
2. Pip install her favorite modules
3. Add her code
4. Test the whole thing locally
5. Create an OpenWhisk action that bundles her exact environment